worker.Dockerfile

```diff
@@ -1,10 +1,30 @@
-FROM jjanzic/docker-python3-opencv
+FROM nvidia/cuda:11.0-runtime-ubuntu18.04 
+
+RUN apt-get update
+RUN apt-get install python3.6 -y
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install python3-pip -y 
 
 ARG FLASK_ENV=production
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
 WORKDIR /app
 COPY . /app
-RUN pip install --upgrade cython && pip install -r worker.requirements.txt && python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'
-CMD python worker.py
+
+RUN pip3 install --upgrade pip
+
+RUN apt-get update
+RUN apt-get install git -y
+
+# Have to install numpy before upgrading cython, which is why these are not 
+# in the worker.requirements.txt file
+RUN pip3 install numpy==1.19.1
+RUN pip3 install --upgrade cython
```

Comment:
abrichr 9 days ago Member
Why are these happening here and not in requirements.txt?

Reply:
cython depends on numpy, and cython-bbox in the requirements.txt file depends on cython, therefore they need to be installed in this order. (pip install does not choose the line order of req.txt in installation)

Reply:
Please add a comment to this effect

Reply:
Done

Reply:
https://github.com/MLDSAI/VisionTracing/pull/2/commits/dd6ae97369dd95ea64b2fdbf0fef2edb27fa8f60